src/pathfinder.js

```diff
@@ -81,8 +82,21 @@ class Pathfinder {
     return cheapestPath
   }
 
+  // params -
+  //   sourceLedger
+  //   destinationLedger
+  //   destinationAmount
+  //   destinationAccount (optional)
```

Comment:
the comment is for this function. It used to take ` (sourceLedger, destinationLedger, destinationAmount) `. Now it takes ` (sourceLedger, destinationLedger, destinationAmount) ` or `(params)`. The comment describes the Object `params`.